docs/javascript/development-process.md

```diff
@@ -24,7 +24,7 @@ Out of that he developed the code linter [JSLint](https://www.jslint.com/).
 
 Because this one was too strict, some other people wrote [JSHint](https://jshint.com/).
 
-Mockup uses JSHint with the following [.jshintrc configuration file](https://github.com/plone/mockup/blob/master/mockup/.jshintrc):
+Mockup uses JSHint with the following [.jshintrc configuration file](https://github.com/plone/mockup/blob/c3b64843f913131836a09d6fde50263358d4ec3b/mockup/.jshintrc):
```

Comment:
It appears that it no longer uses JSHint. This should be removed or updated with whatever is being used now.

Reply:
I suggest to keep it as possible for Plone 5, mark it as outdated and put a friendly note.
-> Issue https://github.com/plone/training/issues/611

Reply:
That's a good plan. OK to ignore this suggestion and keep your change, so that we can archive it for Plone 5 Trainings.